ansible/roles/zabbixdata_sync/templates/zabbix_info.j2

```diff
@@ -0,0 +1 @@
+node_number_onzabbix = {{ host_numberzabbix }}
```

Comment:
can we also list all the nodes from zabbix so we can identify exactly which nodes are missing?